server/apiv2/handlers/keyspace.go

```diff
@@ -0,0 +1,344 @@
+// Copyright 2022 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"github.com/pingcap/kvproto/pkg/keyspacepb"
+	"github.com/tikv/pd/pkg/errs"
+	"github.com/tikv/pd/server"
+	"github.com/tikv/pd/server/apiv2/middlewares"
+	"github.com/tikv/pd/server/keyspace"
+)
+
+// RegisterKeyspace register keyspace related handlers to router paths.
+func RegisterKeyspace(r *gin.RouterGroup) {
+	router := r.Group("keyspaces")
+	router.Use(middlewares.BootstrapChecker())
+	router.POST("", CreateKeyspace)
+	router.GET("", LoadAllKeyspaces)
+	router.GET("/:name", LoadKeyspace)
+	router.PATCH("/:name/updateConfig", UpdateKeyspaceConfig)
```

Comment:
Just curious, do we have existing path with camel case?

Reply:
Not really, the idea of using camel case for action onto resources comes from [GCE's api](https://cloud.google.com/compute/docs/reference/rest/v1/instances/updateNetworkInterface). But it seems APIv1 uses hyphen, for example "/admin/reset-ts".

Reply:
I have proposed an RFC about the PD HTTP v2 API. See https://github.com/tikv/rfcs/pull/88. And if there are conflicts, please comment on the PR.